tutorials/dapt-curation/code/configs/text_semantic_dedupe_config.yaml

```diff
@@ -6,7 +6,7 @@ num_files: 16
 embeddings_save_loc: "embeddings"
 embedding_model_name_or_path: "sentence-transformers/all-MiniLM-L6-v2"
 embedding_batch_size: 128
-write_embeddings_to_disk: false
+write_embeddings_to_disk: true
```

Comment:
Since this closes https://github.com/NVIDIA/NeMo-Curator/issues/505, we can switch this back to true.